src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-revisions-controller.php

```diff
@@ -322,11 +325,19 @@ public function prepare_item_for_response( $post, $request ) {
 			$data['parent'] = (int) $parent->ID;
 		}
 
-		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
-		$data    = $this->add_additional_fields_to_object( $data, $request );
-		$data    = $this->filter_response_by_context( $data, $context );
+		$context             = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data                = $this->add_additional_fields_to_object( $data, $request );
+		$data                = $this->filter_response_by_context( $data, $context );
+		$response            = rest_ensure_response( $data );
+		$links               = array();
+		$resolved_theme_uris = WP_Theme_JSON_Resolver::get_resolved_theme_uris( $theme_json );
```

Comment:
`$theme_json` might be `array()` here which isn't what `get_resolved_theme_uris` is expecting.

Reply:
In which case it'll return the array. There's an instance check in get_resolved_theme_uris.

Do you reckon I should add a check outside the function?

Reply:
Aha, gotcha. I suppose it's ok. Maybe default `$theme_json` to `null` instead of `array()` so it's clearer that it's not an actual value.

Reply:
Good idea. Ta.